data/json/vehicles/trains.json

```diff
@@ -249,5 +249,142 @@
+      { "x": 3, "y": 1, "parts": [ "hdframe_horizontal_2", "halfboard_horizontal_2", "plating_steel" ] },
+      { "x": 3, "y": 2, "parts": [ "hdframe_horizontal_2", "halfboard_horizontal_2", "headlight", "plating_steel" ] },
+      { "x": 3, "y": 3, "parts": [ "hdframe_ne", "halfboard_ne", "plating_steel" ] },
+      { "x": -1, "y": -2, "parts": [ "hdframe_vertical_2", "board_vertical_left" ] },
+      { "x": -1, "y": -1, "parts": [ "hdframe_cross", "aisle_horizontal", "roof" ] },
+      { "x": -1, "y": 0, "parts": [ "hdframe_cross", "aisle_horizontal", "roof" ] },
+      { "x": -1, "y": 1, "parts": [ "hdframe_cross", "aisle_vertical", "roof" ] },
+      { "x": -1, "y": 2, "parts": [ "hdframe_cross", "aisle_horizontal", "roof" ] },
+      { "x": -1, "y": 3, "parts": [ "hdframe_cross", "aisle_horizontal", "roof" ] },
+      { "x": -1, "y": 4, "parts": [ "hdframe_vertical_2", "board_vertical_right" ] },
+      { "x": -2, "y": -2, "parts": [ "hdframe_vertical_2", "board_vertical_T_left" ] },
+      { "x": -2, "y": -1, "parts": [ "hdframe_cross", "board_horizontal", "roof" ] },
+      { "x": -2, "y": 0, "parts": [ "hdframe_cross", "stowboard_horizontal", "roof" ] },
+      { "x": -2, "y": 1, "parts": [ "hdframe_cross", "door_sliding", "roof" ] },
+      { "x": -2, "y": 2, "parts": [ "hdframe_cross", "stowboard_horizontal", "roof" ] },
+      { "x": -2, "y": 3, "parts": [ "hdframe_cross", "board_horizontal", "roof" ] },
+      { "x": -2, "y": 4, "parts": [ "hdframe_vertical_2", "board_vertical_T_right" ] },
+      { "x": -3, "y": -2, "parts": [ "hdframe_vertical_2", "door_sliding", "door_motor" ] },
+      { "x": -3, "y": -1, "parts": [ "hdframe_cross", "rail_wheel", "aisle_horizontal", "roof" ] },
+      { "x": -3, "y": 0, "parts": [ "hdframe_cross", "aisle_horizontal", "roof" ] },
+      { "x": -3, "y": 1, "parts": [ "hdframe_cross", "aisle_vertical", "roof" ] },
+      { "x": -3, "y": 2, "parts": [ "hdframe_cross", "aisle_horizontal", "roof" ] },
+      { "x": -3, "y": 3, "parts": [ "hdframe_cross", "rail_wheel", "aisle_horizontal", "roof" ] },
+      { "x": -3, "y": 4, "parts": [ "hdframe_vertical_2", "door_sliding", "door_motor" ] },
+      { "x": -4, "y": -2, "parts": [ "hdframe_vertical_2", "board_vertical_T_left" ] },
+      { "x": -4, "y": -1, "parts": [ "hdframe_cross", "board_horizontal", "roof" ] },
+      { "x": -4, "y": 0, "parts": [ "hdframe_cross", "board_ne", "roof" ] },
+      { "x": -4, "y": 1, "parts": [ "hdframe_cross", "lit_aisle_vertical", "roof" ] },
+      { "x": -4, "y": 2, "parts": [ "hdframe_cross", "board_nw", "roof" ] },
+      { "x": -4, "y": 3, "parts": [ "hdframe_cross", "board_horizontal", "roof" ] },
+      { "x": -4, "y": 4, "parts": [ "hdframe_vertical_2", "board_vertical_T_right" ] },
+      { "x": -5, "y": -2, "parts": [ "hdframe_vertical_2", "board_vertical_left" ] },
+      { "x": -5, "y": -1, "parts": [ "hdframe_cross", "engine_electric_train", "spring_plate", "roof" ] },
```

Comment:
This should probably have 3 motors, to match the vehicle it is inspired by.

Reply:
I wasn't 100% sure about if the output listed on wikipedia is one of those motors or all 3 together

Reply:
but if I did that do you think it would be okay to have a fourth large storage battery in the "engine rooms"? I have 3 in there right now but only because I was thinking about aesthetics by making the fourth tile have an engine instead

Reply:
Yeah, I think it would be fine.